src/apispec/ext/marshmallow/__init__.py

```diff
@@ -91,7 +93,7 @@ def init_spec(self, spec):
         super().init_spec(spec)
         self.spec = spec
         self.openapi_version = spec.openapi_version
-        self.openapi = OpenAPIConverter(
+        self.openapi = self.Converter(
```

Comment:
Maybe rename this to `self.converter`? No big deal, though.

Reply:
I picked it from your comment here: https://github.com/marshmallow-code/apispec/pull/488#issuecomment-526842419

I figured capitalized could make sense since it is a class.

Otherwise, you'd subclass it like this:

```py
class MyMarshmallowPlugin(MarshmallowPlugin):
    class converter(MarshmallowPlugin.converter):  # <-- lowercase
        def property2parameter(self, ...):
            ...
```